### PR TITLE
feat(visibility): add visibilities

### PR DESCRIPTION
**Pull request by open-dynaMIX**

### CODE REVIEW

File: alexandria/settings.py [hunk 1]
```diff
@@ -143,7 +153,7 @@ def parse_languages(languages):
     "DEFAULT_AUTHENTICATION_CLASSES": (
         "mozilla_django_oidc.contrib.drf.OIDCAuthentication",
     ),
-    "DEFAULT_PERMISSION_CLASSES": ("rest_framework.permissions.IsAuthenticated",),
+    "DEFAULT_PERMISSION_CLASSES": ("rest_framework.permissions.AllowAny",),
```
Comment (open-dynaMIX) [inline alexandria/settings.py:156]: This is just a temporary change until the permissions are implmented.

File: alexandria/core/apps.py [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
 
 
 class DefaultConfig(AppConfig):
     name = "alexandria.core"
     label = "alexandria_core"
+
+    def ready(self):
+        from .models import VisibilityMixin
+
+        # to avoid recursive import error, load extension classes
```
Comment (winged) [inline alexandria/core/apps.py:13]: Comment should be before the import above

File: alexandria/core/visibilities.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        return queryset
+
+
+class Any(BaseVisibility):
+    """No restrictions, all models are exposed."""
+
+    pass
+
+
+class OwnAndAdmin(BaseVisibility):
```
Comment (winged) [inline alexandria/core/visibilities.py:80]: Might need a docstring for a quick explanation

File: alexandria/settings.py (hunk 1 above)
Comment (winged) [reply alexandria/settings.py:156]: In this case, please add a `# TODO ` comment in the code :)

File: alexandria/core/visibilities.py (hunk 3 above)
Comment (open-dynaMIX) [reply alexandria/core/visibilities.py:80]: Done

File: alexandria/settings.py (hunk 1 above)
Comment (open-dynaMIX) [reply alexandria/settings.py]: Done

File: alexandria/core/apps.py (hunk 2 above)
Comment (open-dynaMIX) [reply alexandria/core/apps.py:11]: Changed. Beware: This was an exact copy/paste from caluma and emeis.

**Pull request merged**